### PR TITLE
Update architecture.yml

### PR DESCRIPTION
**Pull request by ivanproskuryakov**

### CODE REVIEW

File: data/architecture.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ questions:
     -
         question: 'What is the Symfony mail service transport parameter name?'
         answers:
-            - {value: mailer.transport,   correct: true}
+            - {value: swiftmailer.transport,   correct: true}
```
Comment (mickaelandrieu) [inline data/architecture.yml]: it's mailer_transport to be honest... can you fix it ?

Comment (ivanproskuryakov) [inline data/architecture.yml]: @mickaelandrieu 

parameter name is:
```yml
transport: '%mailer_transport%'
```
http://symfony.com/doc/current/cookbook/email/email.html


But the service name is "swiftmailer"
https://github.com/symfony/swiftmailer-bundle/blob/6b7316b3471fe28c35c047b4bf17a2ff946f5f66/Resources/config/swiftmailer.xml#L33-L35

Comment (ivanproskuryakov) [inline data/architecture.yml]: But I'm still not sure about this question and answer

Comment (ivanproskuryakov) [inline data/architecture.yml]: hate to be annoying person, also this one https://github.com/ProPheT777/certificationy-data/pull/25

Comment (mickaelandrieu) [inline data/architecture.yml]: In this question we ask the "service transport *parameter* name", so the actual answer is good.

Anyway, thank you for your report: I only learn something every time a report on this repository is done ^_^

**Pull request closed without merging**